src/libretro/LibretroTranslator.cpp

```diff
@@ -334,171 +309,145 @@ const char* LibretroTranslator::GetKeyName(uint32_t code)
 {
   switch (code)
   {
-    case 0x0008: return "backspace";
```

Comment:
Pushed an update. Still not sure if that's what you wanted but there's currently nothing that would use the keynames from libretro api. Key names is only used once for logging a debug message. Not sure if it's worth the effort to add another mapping table from libretro to string.